src/bot.rs

```diff
@@ -626,25 +626,9 @@ impl Bot {
     pub fn send_invoice(
```

Comment:
Remove the `#[allow(clippy::too_many_arguments)]` attribute above as it's no longer needed.